deploy-deps.sh

```diff
@@ -37,13 +37,36 @@ check_req(){
 }
 
 deploy() {
+    test_pvc_binding
     deploy_cert_manager
     kubectl apply -k "${script_path}/dependencies/cluster-issuer"
     deploy_tekton
     deploy_keycloak
     deploy_registry
 }
 
+test_pvc_binding(){
+    PVC_RESOURCES="${script_path}/dependencies/pre-deployment-pvc-binding"
+    PVC_NAME=test-pvc
+    PVC_NAMESPACE=test-pvc-ns
+
+    kubectl apply -k "$PVC_RESOURCES"
+
+    timeout_start=$(date +%s)
+    while [[ $(kubectl get pvc "$PVC_NAME" -n "$PVC_NAMESPACE" -o jsonpath='{.status.phase}') != "Bound" ]]; do
+        if [[ "$(($(date +%s) - timeout_start))" -ge 60 ]]
+        then
+            echo "Couldn't bound `$PVC_NAME` PVC within the expected time. Exiting."
```

Comment:
`$PVC_NAME` is not being printed here.

Reply:
I think we should mention the path to where the PVC is defined and the storage class it tries to use.
Ultimately, what's important is that the user knows which storage class they need to be able to create PVCs for.

Reply:
(I edited the comment above)